plugins/foreman_discovery/4.1/index.md

```diff
@@ -88,15 +95,15 @@ plugin:
     <td>= 1.9</td>
     <td>4.0.0</td>
     <td>1.0.2</td>
-    <td>2.1.1</td>
+    <td>2.1</td>
     <td>0.0.2</td>
   </tr>
   <tr>
     <td>= 1.10</td>
-    <td>4.1.1</td>
-    <td>1.0.3</td>
-    <td>3.0.2</td>
-    <td>0.0.2</td>
+    <td>{{page.pluginver}}</td>
+    <td>{{page.proxyver}}</td>
+    <td>{{page.imgver}}</td>
+    <td>{{page.cliver}}</td>
```

Comment:
@stbenjam corrected the typo, also introduced some options here so URL links and commands do not get incorrect once we bump the versions.